docs/architecture/blazor-for-web-forms-developers/pages-and-routing.md

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
pages-and-routing.md -> pages-routing.md

We don't use small unnecessary words: https://review.docs.microsoft.com/en-us/help/contribute/file-path-guidelines?branch=master#name-your-files